docs/documentation/cn/extending-doris/doris_storage_optimization.md

```diff
@@ -0,0 +1,198 @@
+# Doris存储文件格式优化 #

```

Comment:
you should add this document to path "docs/documentation/cn/internal"

Reply:
OK